DESIGN.md

```diff
@@ -0,0 +1,63 @@
+# ODD
+
+## Introduction
+
+ODD stands for Open Distributed Data. We want our data accessible from anywhere and distributed, meaning that it can live partially or fully in multiple places. The ODD SDK does this through the [WNFS protocol](https://github.com/wnfs-wg).
+
+Ideally as much as possible of this data lives on the devices of the owner of the data and communication of this data would be done in a decentralised peer-to-peer fashion. The ODD stack uses [UCAN](https://github.com/ucan-wg) to provide devices and users with the necessary authorisation. All of this should be usable offline.
```

Comment:
Move the UCAN reference up one paragraph, and reduce technical details up-front, to emphasize the properties ("what") of the system, rather than the method ("how")?